src/app/common/stores/applications/SsvWeb/Operator.store.ts

```diff
@@ -442,22 +383,18 @@ class OperatorStore extends BaseStore {
         const { id, fee } = await getOperator(operator.id);
         return { id, fee };
       },
-      callbackAfterExecution: this.myAccountStore.refreshOperatorsAndClusters,
       prevState: {
         id: operator.id,
         fee: operator.fee,
       },
+      dispatch,
     });
   }
 
-  /**
-   * Remove Operator
-   * @param operatorId
-   * @param isContractWallet
-   */
-  async removeOperator({ operatorId, isContractWallet }: {
+  async removeOperator({ operatorId, isContractWallet, dispatch }: {
```

Comment:
I think U can move the another contract manipulations from Operator.store.ts to operatorContract.service.ts